src/main/java/robot/subsystems/drivetrain/pure_pursuit/Constants.java

```diff
@@ -24,9 +24,17 @@
     public final static double kA = 0.002;
     public final static double kP = 0.01;
 
+    // fall control
+    public final static double Kj = 0;
+    public final static double Kam = 0;
+    public final static double kXdiffrence = 0;// the constant for the mistake in the x distance between roll axis to Center mass
+    public final static double Kmistake = 0;
+
     public final static double STOP_SPEED_THRESH = 0.1; //the speed the robot could stop at the end of the path.
 
     public final static double CYCLE_TIME = 0.02; //cycle time of the roborio
 
     public final static double ROBOT_WIDTH = 0.6; //width of the robot
+
+    public final static double g = 10;// gravity
```

Comment:
If it is a constant, it should be `ALL_CAPS_WITH_UNDERSCORES`. Else, it should be `camelCase`.
Also, with only `10`? Give it a few more digits. If it only `10`, make it `int` instead of `double`.

Reply:
fixed